src/styles/theme.ts

```diff
@@ -104,7 +105,7 @@ export const lightTheme: ThemeAttributes = {
   mode: themes.LIGHT,
   BACKGROUND: colors.WHITE,
   AVATAR_BORDER: colors.PURPLE,
-  INPUT_BACKGROUND: colors.GREY_LIGHT,
+  INPUT_BACKGROUND: colors.WHITE,
```

Comment:
Why did you change this ?

Reply:
Because isn't the good color for this theme